src/meta/src/hummock/manager/transaction.rs

```diff
@@ -145,14 +149,41 @@ impl<'a> HummockVersionTransaction<'a> {
 
         // Append SSTs to a new version.
         for (compaction_group_id, inserted_table_infos) in commit_sstables {
+            let mut accumulated_size = 0;
+            let mut ssts = vec![];
+            let (_, config) = compaction_groups
+                .get(&compaction_group_id)
+                .unwrap_or_else(|| {
+                    panic!(
+                        "compaction group {:?} not found in compaction_groups",
+                        compaction_group_id
+                    )
+                });
+            let sub_level_size_limit = config.sub_level_max_compaction_bytes * 2;
+
             let group_deltas = &mut new_version_delta
                 .group_deltas
                 .entry(compaction_group_id)
                 .or_default()
                 .group_deltas;
-            let group_delta = GroupDelta::NewL0SubLevel(inserted_table_infos);
 
-            group_deltas.push(group_delta);
+            for sst in inserted_table_infos {
```

Comment:
Could you explain how the SSTables maintain a specific chronological order in inserted_table_infos?

Reply:
+1

Reply:
I also wonder whether the current implementation is correct because I remember that newer sst comes first in `inserted_table_infos `.